common/txmgr/confirmer.go

```diff
@@ -128,6 +128,8 @@ type Confirmer[
 	ks               txmgrtypes.KeyStore[ADDR, CHAIN_ID, SEQ]
 	enabledAddresses []ADDR
 
+	tracker *Tracker[CHAIN_ID, ADDR, TX_HASH, BLOCK_HASH, R, SEQ, FEE]
```

Comment:
Out of curiosity, can you like typedef the array of generic types used here in Go? This way you don't have to keep repeating this list everywhere in the code.

Reply:
Not that I know of in Go.